### PR TITLE
✨ Replace VELO with Extra Finance

### PR DESCRIPTION
**Pull request by sebipap**

### CODE REVIEW

File: components/ExtraFinance/index.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      <a href={EXTRA_EXA_LENDING_POOL_URL} target="_blank" rel="noreferrer noopener">
+        <Button variant="outlined">
+          <Box display="flex" gap={0.5} alignItems="center">
+            <Avatar
+              alt="EXRA Token"
+              src={`/img/assets/EXTRA.svg`}
+              sx={{ width: 16, height: 16, fontSize: 10, borderColor: 'transparent' }}
+            />
+            <Typography fontSize={14} fontWeight={700} noWrap>
+              {apr !== undefined ? `${apr} APR` : t('Supply EXA')}
```
Comment (jgalat) [inline components/ExtraFinance/index.tsx:29]: Should this be _Deposit EXA_ instead of _Supply_?

File: hooks/useExtra.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    address: LENDING_CONTRACT_ADDRESS,
+    chainId: optimism.id,
+  });
+  if (!reserves) return undefined;
+  const [{ borrowingRate, totalBorrows, totalLiquidity }] = reserves;
+  const utilizationRate = (totalBorrows * WAD) / totalLiquidity;
+  return (borrowingRate * utilizationRate) / WAD;
+};
+
+const aprtoAPY = (apr: number, interval: number) =>
```
Comment (jgalat) [inline hooks/useExtra.ts:23]: Use bigint for this?

**Review by jgalat — APPROVED** [review]
LGTM, minor comments

**Comment by cruzdanilo** [comment]
@itofarina please, review the apr/apy logic

**Review by jgalat — APPROVED** [review]

**Review by JuampiRombola — APPROVED** [review]

**Pull request merged**